### ISSUE TITLE
Improved partial for task (and supply) listings

### ISSUE THREAD

**Issue by dottorer** [lale-help/lale-help#239]
Based on user suggestions, across the system we should update the partial used for task list rendering in order to convey data more directly and intuitively.

Here is a design concept:
![image](https://cloud.githubusercontent.com/assets/14217285/13979945/fb294186-f0db-11e5-94cf-96235714ca4a.png)

Improvements over today:
* clearer date visibility, more of a sequential calendar view
* task state (new/urgent/filled/completed) visible in date field color, not the dot. 
* task title more readable (larger font)
* current assignees listed

Also, if possible in the real estate we have, we should add the weekday short indicator to the date object like so:
![image](https://cloud.githubusercontent.com/assets/14217285/13989008/cad3c7e4-f10e-11e5-94a3-c5f0c31db5a8.png)

**Comment by dottorer** [comment]
This would be an improvement useful for sequential tasks in a series as in #29 .

**Comment by dottorer** [comment]
Assigning to @phillipoertel to discuss in the context task projects.

**Comment by dottorer** [comment]
Assigning to @kristysnet to work on in parallel of @phillipoertel 's work on #29 so we can finish it in the next iteration by April 8th. We can discuss in our dev sync today.

**Comment by dottorer** [comment]
@kristysnet Awesome first cut at the new task list! We merged and integrated this in master together with @phillipoertel code for projects today and I tested it a bit. 

Here are some small nits to address:

![image](https://cloud.githubusercontent.com/assets/14217285/14285505/1c8d39de-fb4c-11e5-92a9-ec061eb71178.png)

1. The task title should align on the top of the task date badge
2. The day-of-week is not doing i18n yet and needs to pick up German day wording
3. The assignee initials should be spaced a tiny bit.
4. The full task view should now also show the day-of-week
![image](https://cloud.githubusercontent.com/assets/14217285/14285579/601fe6ec-fb4c-11e5-8a7e-02fca046fd62.png)

Otherwise I think this look sweet! For any changes please remember to rebase off of the latest master to gets Phillips last code changes...

**Comment by dottorer** [comment]
@kristysnet Very nice. Few things:

1. For supplies, currently we only allow 1 assigned helper, but we display more than one on the listing:
![image](https://cloud.githubusercontent.com/assets/14217285/14314631/ab35d4f0-fbf8-11e5-8960-9734d22f04b1.png)
2. Supply badge in detail view does not show day of week yet:
![image](https://cloud.githubusercontent.com/assets/14217285/14314680/018de9b4-fbf9-11e5-9808-f584b7529590.png)
3. The dimensions (width, height, font size) of the badge in the listing are perfect, however in the detail view they seem a bit more long. Can we make the detail big badge the same dimensions as in the listing (while keeping it a bit bigger of course).

Thanks!

**Comment by dottorer** [comment]
Hi @kristysnet ... possibly last nit change for this sweet feature...

**1. task list item does not adjust width on iPad / iOS**
![image](https://cloud.githubusercontent.com/assets/14217285/14424412/2c06a104-ffe1-11e5-9e3f-35fc531f0b7e.png)

**Comment by dottorer** [comment]
Looking very nice now! Thanks @kristysnet
